bin/GameIndex.dbf

```diff
@@ -34770,6 +34770,12 @@ Region = NTSC-U
 Compat = 5
 eeClampMode = 3 // Objects appear in wrong places without it 
 vuClampMode = 2 // Fixes water reflection
+
+[patches = 2F56CBC9]
+ comment=patch by Prafull
+// avoid cave hang at volk city
+patch=1,EE,00305bcc,word,00000000
```

Comment:
Also add extra whitespace for the 3 lines.

Reply:
You're so picky 😛

Reply:
OCD overload. Btw I did the same for my PR a few days ago